readme.md

```diff
@@ -302,23 +313,135 @@ Sets `options.method` to the method name and makes a request.
 
 #### Instances
 
-#### got.create([options])
+#### got.extend([options])
 
-Configure a new `got` instance with default `options`:
+Configure a new `got` instance with default `options` and custom `baseUrl` (optional):
 
 ```js
 (async () => {
```

Comment:
This example is a bit unclear if you're not familiar with `httpbin.org` and knowing that this endpoint will reflect headers back in the body. Maybe something like this would help illustrate: 

```js
const client = got.extend({
	baseUrl: 'https://example.com',
	headers: {
		'x-unicorn': 'rainbow'
	}
});

client.get('/demo')

/* HTTP Request => 
 * GET /demo HTTP/1.1
 * Host: example.com
 * x-unicorn: rainbow
 */
```

Reply:
Oh wait, you mean users may not be familiar with `httpbin.org`? Then yes too. I'll replace that.